src/borg/archiver.py

```diff
@@ -1608,7 +1610,8 @@ def do_recreate(self, args, repository, manifest, key, cache):
                 if recreater.is_temporary_archive(name):
                     continue
                 print('Processing', name)
-                recreater.recreate(name, args.comment)
+                if not recreater.recreate(name, args.comment):
+                    logger.info('Skipped archive %s: Nothing to do. Archive was not processed.', name)
```

Comment:
For mass-recreate this would kinda be expected. E.g. rechunking archives in a mixed repository — it's expected that newer archives would not be processed.